docker/qchem.dockerfile

```diff
@@ -17,7 +17,6 @@ FROM pennylane/base:latest
 # Update and install Qchem
 RUN DEBIAN_FRONTEND="noninteractive" apt-get install tzdata # need to perform this again
 RUN apt-get update \
-    && apt-get -y install --no-install-recommends make git openbabel \
```

Comment:
@soranjh sorry I missed this one! But `openbabel` was only one of the packages being installed, do we still need `make` and `git`? If so, we will need to revert the line to
```docker
    && apt-get -y install --no-install-recommends make git \
```

Reply:
Hey Josh. I am actually making my final PR to bring qchem inside pennylane and remove the need for installation. I can create a small PR just to retrieve this line during the few days transition period until that PR gets merged.